### PR TITLE
style fixes #139 #145

### LINKED ISSUE #139: 6062 Single teaser image size (too high)
**URL:** https://dev.liqd.net/de/academy-landing-page/
**user:** cms editor
**expected behaviour:** Block crops image to the right size
**behaviour:** Higher images don't get cropped to the right size
**important screensize:** mobile and desktop
**device & browser:** 
**Comment/Question:** 

Screenshot?

image

Design: 

image

### LINKED ISSUE #144: #6064 [liqd.net] second-level navigation
**URL:** https://dev.liqd.net/en/academy-landing-page/
**user:** *all*
**important screensize:** *Desktop*
**device & browser:** *Safari Version 15.6.1 (17613.3.9.1.16) and Chrome Version 104.0.5112.101 (Official Build) (x86_64)*
**Comment/Question:** *I don't know how specific editing the underline can be, so in case its not possible, ignore this message. Could we change a bit the spacing? Currently the line extends beyond the word a little bit, and when one word is selected, there is one giant line from one word in the menu to the other (Screenshot below'). I would also try to make a bit closer vertically too. Thanks!*

Screenshot-2022-09-08-at-11 14 20

Screenshot 2022-09-08 at 11 33 33

### LINKED ISSUE #145: #6064 [liqd.net] second-level navigation
**URL:** https://dev.liqd.net/en/academy-landing-page/
**user:** *all*
**important screensize:** *Desktop*
**device & browser:** *Safari Version 15.6.1 (17613.3.9.1.16) and Chrome Version 104.0.5112.101 (Official Build) (x86_64)*
**Comment/Question:** *Shall we use the white gradient behind the menu here as well?*

Screenshot 2022-09-08 at 11 39 27
Screenshot 2022-09-08 at 11 40 50

### PR DESCRIPTION
**Pull request by khamui**
fixes #145 
fixes #139 

fixes #144

### CODE REVIEW

File: apps/core/static/scss/_blocks.scss [hunk 1]
```diff
@@ -159,6 +159,8 @@
 
 .block--single_teaser img {
     width: 100%;
+    aspect-ratio: 3/2;
```
Comment (fuzzylogic2000) [inline apps/core/static/scss/_blocks.scss:162]: @philli-m Is that fine? It looks good. :)

**Review by fuzzylogic2000 — APPROVED** [review]
Nice! It does look fine now. Just a qestion for @philli-m as I haven't seen that aspect-ratio thing and the object-fit yet.

File: apps/core/static/scss/_blocks.scss (hunk 1 above)
Comment (fuzzylogic2000) [reply apps/core/static/scss/_blocks.scss:162]: Ok, it doesn't work in IE :rofl: , but apart from that seems to be ok.

Comment (khamui) [reply apps/core/static/scss/_blocks.scss:162]: oh, did not check IE. Do we support IE?

Comment (philli-m) [reply apps/core/static/scss/_blocks.scss:162]: yesish but I think we have other problems then this so I would say it's fine!

File: apps/core/static/scss/_header.scss [hunk 2]
```diff
@@ -249,3 +249,11 @@ button.header__bar__toggle {
...
     }
 }
+
+.navbar-expand-md .navbar-nav .nav-link {
+    padding-right: $spacer * 0.1;
+    padding-left: $spacer * 0.1;
+    margin-right: $spacer * 0.3;
+    margin-left: $spacer * 0.3;
+    line-height: 1.1;
+}
```
Comment (philli-m) [inline apps/core/static/scss/_header.scss:259]: this seems to overwrite styling that looks the same, is there a reason to do this?

**Review by philli-m — COMMENTED** [review]
just question about overwritting nav-link styling

File: apps/core/static/scss/_header.scss (hunk 2 above)
Comment (khamui) [reply apps/core/static/scss/_header.scss:259]: ah sorry, forgot to mention the issue it is fixing: #144

Comment (khamui) [reply apps/core/static/scss/_header.scss]: @philli-m okay moved and added comment.

**Review by philli-m — APPROVED** [review]
coolio!

**Pull request merged**